lib/composer/wikia/fluent-sql-php/tests/FluentSql/NonNumericLimitOffsetTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Test non-numeric LIMIT and OFFSET
+ *
+ * @author Mix <mix@wikia.com>
+ */
+
+namespace FluentSql;
+
+class NonNumericLimitOffsetTest extends FluentSqlTestBase {
+
+	/**
+	 * numeric LIMIT
+	 */
+	public function test1() {
```

Comment:
That was the naming scheme used here https://github.com/Wikia/fluent-sql-php/blob/master/tests/FluentSql/ComplexQueryTest.php - but yeah, names could be more descriptive. This needs to be addressed as a contribution to https://github.com/Wikia/fluent-sql-php